packages/theme-predictive-search-component/__tests__/theme-predictive-search-component.test.js

```diff
@@ -0,0 +1,540 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable no-unused-vars */
+
+/**
+ * TODO
+ * - Add test for kill()
+ * - Add test for body event to close predictive search
+ */
+
+import xhrMock from "xhr-mock";
+import PredictiveSearchComponent from "../src/theme-predictive-search-component";
+import searchAsYouTypeTheCallingFixture from "../__fixtures__/search_as_you_type_the_calling.json";
+
+function defaultResultTemplateFct(data) {
+  return `
+    <ul>
+      ${data.products.map(
+        item => `
```

Comment:
nit: could these be `product` instead of `item` to keep consistency with the README?